.gitlab-ci.yml

```
@@ -2529,14 +2515,10 @@ testkitchen_cleanup_azure-a7:
 cleanup_kitchen_functional_test:
   <<: *kitchen_cleanup_azure_common
   rules:
-    - when: always
+    - when: manual
       allow_failure: true
   stage: functional_test_cleanup
-  needs:
-    - kitchen_centos_security_agent
-    - kitchen_debian_security_agent
-    - kitchen_suse_security_agent
-    - kitchen_ubuntu_security_agent
+  needs: []
```

Nit: for consistency, you could add:
```suggestion
  needs: [ "tests_ebpf" ]
```
so that the cleanup step is available under the same conditions as the functional tests themselves.